src/Compilers/CSharp/Test/Semantic/Semantics/TopLevelStatementsTests.cs

```diff
@@ -5158,15 +5159,16 @@ static async Task Main()
             var comp = CreateCompilation(text, options: TestOptions.DebugExe, parseOptions: DefaultParseOptions);
 
             comp.VerifyDiagnostics(
+                // (7,7): error CS0260: Missing partial modifier on declaration of type 'Program'; another partial declaration of this type exists
+                // class Program
+                Diagnostic(ErrorCode.ERR_MissingPartial, "Program").WithArguments("Program").WithLocation(7, 7),
                 // (9,17): warning CS7022: The entry point of the program is global code; ignoring 'Program.Main(string[])' entry point.
                 //     static void Main(string[] args)
                 Diagnostic(ErrorCode.WRN_MainIgnored, "Main").WithArguments("Program.Main(string[])").WithLocation(9, 17),
                 // (14,23): warning CS7022: The entry point of the program is global code; ignoring 'Program.Main()' entry point.
                 //     static async Task Main()
                 Diagnostic(ErrorCode.WRN_MainIgnored, "Main").WithArguments("Program.Main()").WithLocation(14, 23)
                 );
-
-            CompileAndVerify(comp, expectedOutput: "Hi!");
```

Comment:
>CompileAndVerify(comp, expectedOutput: "Hi!"); [](http://example.com/codeflow?start=12&length=46)

It looks like this test was testing interesting success scenario. I think we should keep it as such. The class name isn't important, so renaming it would be more appropriate.